docs/changelog.rst

```diff
@@ -4,6 +4,41 @@ CHANGELOG
 Raster Vision 0.13
 -------------------
 
+This release presents a major jump in Raster Vision's power and flexibility. The most significant changes are:
+
+Support arbitrary models and loss functions (`#985 <https://github.com/azavea/raster-vision/pull/985>`_, `#992 <https://github.com/azavea/raster-vision/pull/992>`_)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Raster Vision is no longer restricted to using the built in models and loss functions. It is now possible to import models and loss functions from a GitHub repo or a URI or a zip file as long as they interface correctly with RV's learner code. This means that you can now easily swap models in your existing training pipelines, allowing you to take advantage of the latest models or to make customizations that help with your specific task; all with minimal changes.
+
+This is made possible by PyTorch's ``hub`` module.
+
+Currently not supported for Object Detection.
+
+Support for multiband images (even with Transfer Learning) (`#972 <https://github.com/azavea/raster-vision/pull/972>`_)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is now possible to train on imagery with more than 3 channels. Raster Vision automatically modifies the model to be able to accept more than 3 channels. If using pretrained models, the pre-learned weights are retained.
+
+The model modification cannot be performed automatically when using an external model. But as long as the external model supports multiband inputs, it will work correctly with RV.
+
+Currently only supported for Semantic Segmentation.
```

Comment:
I'm curious about why this is only supported for semantic segmentation. It seems like only the first layer has to be modified, and this is the same across computer vision tasks?

Reply:
It is indeed possible to do this for other tasks. It currently only works for SS because it was added under https://github.com/azavea/raster-vision/issues/961 which was SS specific.